man/fi_info.1.md

```diff
@@ -92,28 +92,184 @@ the fi_info structure, see fi_getinfo(3).
 # USAGE EXAMPLES
 
 ```
-$ fi_info -n 30.0.11.1 -p usnic -t FI_EP_DGRAM
+$ fi_info -p verbs -t FI_EP_DGRAM
 ```
 
-This will respond with all fabric interfaces that can reach address 30.0.11.1
-using endpoint type FI_EP_DGRAM with the usNIC provider.
+This will respond with all fabric interfaces that use endpoint type
+FI_EP_DGRAM with the verbs provider.
+
+```
+fi_info \-c FI_MSG|FI_READ|FI_RMA
```

Comment:
The capabilities need to be string quoted.  Typing the above directly onto a command line will produce an error, since FI_READ will be interpreted as a command with the output of fi_info piped into it.  Similar comments apply below.